text/0000-sync-networked-vars.md

```diff
@@ -0,0 +1,111 @@
+- Feature Name: `synced_networked_var`
+- Start Date: 2021-02-12
+- RFC PR: [Unity-Technologies/com.unity.multiplayer.rfcs#0000](https://github.com/Unity-Technologies/com.unity.multiplayer.rfcs/pull/0000)
+- Issue: [Unity-Technologies/com.unity.multiplayer#0000](https://github.com/Unity-Technologies/com.unity.multiplayer/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Make the NetworkedVariables synchronized with the tick system.
+
+We suggest two elements:
+- Changing `NetworkedBehaviour::NetworkedBehaviourUpdate()` so that the function runs at most one per Network Tick. When called more than once per Network Tick, it would simply return.
+- NetworkedVariables carrying two pieces of information with their delta updates:
+    - The local tick at which a variable was modified
+    - The remote tick at which the variable was modified, when the host relays the variable to clients, including the source client    
+
+# Motivation
+[motivation]: #motivation
+
+In a client-server topology, any client and the server can modify NetworkedVariables. The server receives the clients' changes and send them back to all clients. In order to retain generality, we'll assume that any variable can be modified by anyone and that the server can also modify the variable as they are received and sent. For example, we could have server-driven physics over client-driven character movement.
+
+We would like the game code on the client machine to be able to pick any interpolation system it desires for character movement. We'd like the game code to be able to pick client- or server-authoritative mechanics as it see fit. In order to allow this, we believe that each NetworkedVariable should know:
+- When was I modified by a client?
+- When did this update travel through the server?
+- Do I have more recent local information about my value?
+
+This will enable better client-side interpolation and lag compensation, in the future, and provide flexibility for systems like snapshotting. 
+
+This will enable the client to tell the difference between receiving its own updates back and receiving updates back with server-side adjustments for things like physics.
+
+This will keep the overall bandwidth usage in check, preventing frequent updates for NetworkedVariables.
```

Comment:
_From: This will enable better client-side interpolation and lag compensation, in the future, and provide flexibility for systems like snapshotting._

**To:** This will enable better client-side interpolation, lag compensation, and in the future provide the required flexibility for new systems (i.e. like snapshotting).

_From: This will enable the client to tell the difference between receiving its own updates back and receiving updates back with server-side adjustments for things like physics._

**To:** This provides enough information such that the client can distinguish between its own updates from that of server-side adjustments (i.e. like physics).